index.js

```diff
@@ -139,19 +133,19 @@ const legacyTextDomains = [...unprotectedDomains].join('\n')
 fs.writeFileSync(`${GENERATED_DIR}/trackers-unprotected-temporary.txt`, legacyTextDomains)
 fs.writeFileSync(`${GENERATED_DIR}/trackers-whitelist-temporary.txt`, legacyTextDomains)
 const legacyNaming = {
```

Comment:
might be worth exporting as you did with `platforms.js` because it's both in this file and `import.js`